### DISCUSSION TITLE
Tiered Application. Web Project is too slow

### DISCUSSION THREAD

**Discussion by crazygeek69** [abpframework/abp#8297] [General]
I created a tiered application with ABP 4.2. When I run it Identity Server and HTTPAPI.Host are  loading pretty fast. But Web project taking lot of time to load. Also Roles, Tenants pages taking almost 45 seconds to load. 

But if create the project as "not tiered" it's loading up very fast. Also I am using Docker Image of Redis.

Note: I am running right after create the application using ABP 4.2 CLI

**Comment by amsaighi** [comment]
Hello
Did you launched Redis ?

**Reply by gterdem** [reply]
Check the application logs if there is a related error with reaching to Redis. There may be a mis-configuration of redis port. Default expected exposed port should be 6379

**Comment by TejpallPathak** [comment]
i am still facing this issue.
did anyone found solution?

**Reply by gterdem** [reply]
Make sure redis is up and running if you are using tiered application.

**Reply by TejpallPathak** [reply]
> 

can you please tell me how to do it in windows?

**Reply by gterdem** [reply]
If you are using docker for windows, run the command `docker run --name redis-container -p 6379:6379 -d redis:alpine`.

**Reply by jfaquinojr** [reply]
> If you are using docker for windows, run the command `docker run --name redis-container -p 6379:6379 -d redis:alpine`.

good thing I decided to google this today, instead of giving up on tiered template. I hope this was obvious for peeps just peeping
